src/gui/guiFormSpecMenu.cpp

```diff
@@ -3512,14 +3521,13 @@ void GUIFormSpecMenu::acceptInput(FormspecQuitMode quitmode=quit_mode_no)
 					if (table) {
 						fields[name] = table->checkEvent();
 					}
-				}
-				else if(s.ftype == f_DropDown) {
-					// no dynamic cast possible due to some distributions shipped
-					// without rtti support in irrlicht
+				// No dynamic cast possible for any element due to some
```

Comment:
this was better in its previous location, even if duplicated, because it was next to the relevant code

Reply:
Duplicating it so many times seems really messy, though.  Is there a better way that we could do it?